packages/core/src/react-integration/__tests__/integration-optimistic-legacy-endpoint.web.tsx

```diff
@@ -0,0 +1,358 @@
+import {
```

Comment:
this is copy/paste of existing tests (packages/core/src/react-integration/__tests__/integration-optimistic-endpoint.web.tsx). to validate existing endpoints. The above file is then adapted for new methods